### PR TITLE
fix(AnimeSuge): fix issues by rewriting presence 

### LINKED ISSUE #7912: Animesuge | animesuge.to - changed their domain from .to to .net in some places
### Steps to Reproduce

well at least in Australia because they opened up a new domain and Australia can't access the old .to one

I was linked from the support page to go here.

### Expected Behavior

n/a

### Actual Behavior

n/a

### Page URL

from https://animesuge.to/ to https://animesuge.net/

### Screenshots

_No response_

### Operating System

Windows 11

### Browser and Version

Chrome

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

- Resolves #7912
- Change domain; 
- Change to better performing static pages check, with more details;
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/A/AnimeSuge/presence.ts [hunk 1]
```diff
@@ -39,98 +19,97 @@ presence.on(
...
+			"/contact": {
+				details: "Reading the contacts",
+				state: "Reading",
+				smallImageKey: Assets.Reading,
+			},
+			"/user/settings": { details: "Changing the settings", state: "Changing" },
+			"/user/watchlist": {
+				details: "Looking at their watchlist",
+				state: search.get("folder")
+					? `At folder: ${search.get("folder")}`
```
Comment (DarkVillager) [inline websites/A/AnimeSuge/presence.ts:58]:
```suggestion
					? `In folder: ${search.get("folder")}`
```

File: websites/A/AnimeSuge/presence.ts [hunk 2]
```diff
@@ -39,98 +19,97 @@ presence.on(
...
-			default: {
-				presenceData.details = "Looking at an unknown page";
-				presenceData.state = "Unknown";
-			}
+		for (const [path, data] of Object.entries(pages)) {
+			if (pathname !== "/" && pathname.includes(path))
+				presenceData = { ...presenceData, ...data };
 		}
 	}
+	if (!presenceData.state) presenceData.state = "Searching animes";
```
Comment (DarkVillager) [inline websites/A/AnimeSuge/presence.ts:112]: This was in the original version, not sure if it should be here or not.

File: websites/A/AnimeSuge/metadata.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
 		"en": "AnimeSuge is a free streaming anime website that allows you to watch anime online in English subbed and dubbed.",
 		"vi_VN": "AnimeSuge là trang web phát anime miễn phí cho phép bạn xem anime với phụ đề hoặc thuyết minh Tiếng Anh."
 	},
-	"url": "animesuge.to",
-	"version": "1.1.16",
+	"url": "animesuge.net",
```
Comment (theusaf) [inline websites/A/AnimeSuge/metadata.json:12]: `.to` still exists with full functionality. Use URL list or regex

File: websites/A/AnimeSuge/presence.ts [hunk 4]
```diff
@@ -39,98 +19,97 @@ presence.on(
...
 			presenceData.smallImageKey = Assets.Pause;
 			presenceData.smallImageText = "Paused";
-			delete presenceData.startTimestamp;
-			delete presenceData.endTimestamp;
 		}
 		presenceData.buttons = [
 			{
 				label: "Watch Episode",
-				url: `http://animesuge.to${page}`,
+				url: `http://animesuge.to${pathname}`,
```
Comment (theusaf) [inline websites/A/AnimeSuge/presence.ts:103]: Perhaps this could be replaced by `href` or something similar?

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/AnimeSuge/metadata.json (hunk 3 above)
Comment (DarkVillager) [reply websites/A/AnimeSuge/metadata.json:12]: I tested it, it now works on both.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**